python/semantic_kernel/connectors/ai/azure_ai_inference/azure_ai_inference_prompt_execution_settings.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from pydantic import Field
+
+from semantic_kernel.connectors.ai.prompt_execution_settings import (
+    PromptExecutionSettings,
+)
+
+
+class AzureAIInferencePromptExecutionSettings(PromptExecutionSettings):
+    """Azure AI Inference Prompt Execution Settings."""
+
+    frequency_penalty: float = Field(0.0, ge=0.0, le=1.0)
```

Comment:
Do we need to define defaults for all of these parameters? If the SDK/Service defined their own defaults for these, and supports these being nullable, that would be preferrable.